lib/main.js

```diff
@@ -112,6 +112,7 @@ let load = loader({
           ec2: ec2,
           runaws: runaws,
           lsChecker: lsChecker,
+          credentials: cfg.taskcluster.credentials,
```

Comment:
Hrm, does it improve security somehow?

Reply:
yes, by allowing us to have a different credential which is very locked down for issuing temporary credentials.  This would let us rotate that key without rotating the provisioner's key and vice versa.